modules/installation-osp-kuryr-port-pools.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-user-kuryr.adoc
+// * post_installation_configuration/network-configuration.adoc
+
+[id="installation-osp-kuryr-port-pools_{context}"]
+= Kuryr ports pools
+
+A Kuryr ports pool maintains a number of ports on standby for pod creation.
+
+Keeping ports on standby minimizes pod creation time. Without ports pools,
+Kuryr must explicitly request port creation or deletion every time a pod is
+created or deleted.
+
+The Neutron ports that Kuryr uses are created in subnets that are tied to namespaces. These ports are attached as subports to trunk ports: the {product-title} cluster node ports.
+
+Because Kuryr keeps each namespace in a separate subnet, a separate ports pool
+is maintained for each namespace-worker pair.
+
+Before triggering the installation you can set the following parameters in the
+`cluster-network-03-config.yml` manifest file to configure ports pool behavior:
+
+* The `enablePortPoolsPrepopulation` parameter allows you to enable pool prepopulation,
+  which forces Kuryr to add ports to the pool when it is created, such as when a
+  new host is added, or a new namespace is created. The default value is `false`.
```

Comment:
```suggestion
* The `enablePortPoolsPrepopulation` parameter controls pool prepopulation, which forces Kuryr to add ports to the pool when it is created, such as when a new host is added, or a new namespace is created. The default value is `false`.
```

Does "it" here mean "the pool"? 

Option: if there are only a handful of situations where ports are added, rephrase to something like:

"which forces Kuryr to add ports to the pool when a new host is added, a new namespace is created, or [other thing]. The default value is `false`."

Reply:
Yup, it's pool, this makes sense.